apps/src/codebridge/InfoPanel/MainInstructionsContent.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import {Theme} from '@cdo/apps/lab2/types';
+import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+
+import ValidationStatusIcon from './ValidationStatusIcon';
+
+import moduleStyles from '@codebridge/InfoPanel/styles/validated-instructions.module.scss';
+
+interface MainInstructionsContentProps {
+  instructionsText: string;
+  handleInstructionsTextClick?: (id: string) => void;
+  theme: Theme;
```

Comment:
Is `theme` prop needed?

Reply:
oh good call, it's only needed for the preview component